web/src/utils/url.ts

```diff
@@ -12,7 +13,11 @@ export const getTimeOfInterestFromUrl = (search: string): string => {
     return queryString
   }
 
-  return formatDateInISO(new Date())
+  const currInPST = moment(new Date())
+    .utcOffset(PST_UTC_OFFSET)
```

Comment:
Am I the only one who finds this confusing? Moments documentation didn't clear anything up for me. If we're going to keep moment, maybe add a comment as to what's happening here - something along the lines of "Take the current date, re-stating it in terms of pacific time (the date/time doesn't change, only the timezone). If you're already in PST, nothing changes."